ci-operator/config/openshift/kedacore-keda/openshift-kedacore-keda-main.yaml

```diff
@@ -6,8 +6,34 @@ base_images:
 build_root:
   from_repository: true
 images:
-- from: base
+- build_args:
+  - name: GIT_COMMIT
+    value: dummy-ci-commit-value
+  from: base
   to: custom-metrics-autoscaler
+- build_args:
+  - name: GIT_COMMIT
+    value: dummy-ci-commit-value
+  dockerfile_path: Dockerfile.webhooks
+  to: custom-metrics-autoscaler-webhooks
+- build_args:
+  - name: GIT_COMMIT
+    value: dummy-ci-commit-value
+  dockerfile_path: Dockerfile.adapter
+  to: custom-metrics-autoscaler-adapter
+- dockerfile_literal: |
+    FROM ghcr.io/kedacore/keda-tools:1.20.5
+    COPY . /src
+    RUN chmod 777 -R /src
+    WORKDIR /src
+    RUN mkdir -p /go && chmod 777 -R /go
+    RUN mkdir -p /bin && chmod 777 -R /bin
+    ENV GOCACHE=/src/.cache
+    ENV USE_SUDO=false
+    ENV PATH=/src/helm:/src/:$PATH
+    ENV HELM_INSTALL_DIR=/src
+    RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && chmod 700 get_helm.sh && ./get_helm.sh
```

Comment:
i think it would be better if this dockerfile lived in the repo instead of inlined here.  but you can fix that as a followup.